storage/ndb/src/kernel/blocks/dblqh/DblqhMain.cpp

```diff
@@ -17597,13 +17623,24 @@ void Dblqh::scanLockReleasedLab(Signal* signal,
     else if (scanPtr->m_last_row && !scanPtr->scanLockHold)
     {
       jam();
+      /*
+       * Moz
+       * TODO (Zhao)
```

Comment:
Is it still a TODO?

Reply:
No. As I described below. It seems no need to do something special for pushdown aggregation so far. But I prefer to leave a comment here as a hook in case of any issue in the future